### ISSUE TITLE
path to vtk build is not right in macos build doc.

### ISSUE THREAD

**Issue by maidamai0** [qCring/QuickVtk#48]
The build path of vtk should be **QuickVtk/denpendencies/src** instead of [**QuickVtk/lib/src**](https://github.com/qCring/QuickVtk/blob/master/doc/build/macos/build.md)

**Comment by qCring** [comment]
Yes, you're right. This is due to some project structure changes from the current refactoring. I didn't update the docs yet but this will be fixed soon. Thanks for the hint, I'll keep this issue open until the docs are in sync with the codebase which should happen soon.
